keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -97,34 +130,38 @@ If services (other than the API server could authenticate KSA tokens directly:
     authenticate KSA tokens.
 -   Attempt compatibility with OIDC: common libraries that authenticate OIDC
     tokens should be able to authenticate KSA tokens.
-
-As a stretch goal / consideration:
-
 -   Support authentication when the API server is not directly reachable by the
     relying party.
     -   e.g.: a cloud-based service authenticating an API server that doesn't
         have a public Internet address.
 
 Note that the API server has a very different flow from OIDC with respect to
-generating tokens. As such, our goal is OIDC *compatibility*...
+generating tokens. As such, our goal is OIDC *compatibility*.
 
 ### Non-Goals
 
-...but not OIDC *compliance*.
+Our goal is *not* OIDC *compliance*.
 
-We aren't trying to: - Make the KSA token process fully compliant with OIDC
-specifications. - OIDC includes flows for token acquisition, token exchange,
-getting user data out of tokens, etc. - But we're only interested in the parts
-relevant to *relying parties*, i.e. token authentication. - We don't need to do
+We aren't trying to make the KSA token process fully compliant with OIDC
+specifications. OIDC includes flows for token acquisition, token exchange,
+getting user data out of tokens, etc. But we're only interested in the parts
```

Comment:
Grammar nit: But at the beginning of a sentence.

```suggestion
getting user data out of tokens, etc. We are primarily interested in the parts
```

Reply:
Done.